api/OpenAI.net8.0.cs

```diff
@@ -6052,6 +6074,86 @@ public class ResponseReasoningOptions : IJsonModel<ResponseReasoningOptions>, IP
         public override readonly string ToString();
     }
     [Experimental("OPENAI001")]
+    public class ResponseResult : IJsonModel<ResponseResult>, IPersistableModel<ResponseResult> {
+        public bool? BackgroundModeEnabled { get; set; }
+        public DateTimeOffset CreatedAt { get; set; }
+        public string EndUserId { get; set; }
+        public ResponseError Error { get; set; }
+        public string Id { get; set; }
+        public ResponseIncompleteStatusDetails IncompleteStatusDetails { get; set; }
+        public string Instructions { get; set; }
+        public int? MaxOutputTokenCount { get; set; }
+        public IDictionary<string, string> Metadata { get; }
+        public string Model { get; }
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Object { get; set; }
+        public IList<ResponseItem> OutputItems { get; }
+        public bool ParallelToolCallsEnabled { get; set; }
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Experimental("SCME0001")]
+        public ref JsonPatch Patch { get; }
+        public string PreviousResponseId { get; set; }
+        public ResponseReasoningOptions ReasoningOptions { get; set; }
+        public ResponseServiceTier? ServiceTier { get; set; }
+        public ResponseStatus? Status { get; set; }
+        public float? Temperature { get; set; }
+        public ResponseTextOptions TextOptions { get; set; }
+        public ResponseToolChoice ToolChoice { get; set; }
+        public IList<ResponseTool> Tools { get; }
+        public float? TopP { get; set; }
+        public ResponseTruncationMode? TruncationMode { get; set; }
+        public ResponseTokenUsage Usage { get; set; }
+        public string GetOutputText();
+        protected virtual ResponseResult JsonModelCreateCore(ref Utf8JsonReader reader, ModelReaderWriterOptions options);
+        protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWriterOptions options);
+        public static explicit operator ResponseResult(BinaryData data);
+        public static explicit operator ResponseResult(ClientResult result);
+        protected virtual ResponseResult PersistableModelCreateCore(BinaryData data, ModelReaderWriterOptions options);
+        protected virtual BinaryData PersistableModelWriteCore(ModelReaderWriterOptions options);
+    }
```

Comment:
This is the other type affected by the PR that Shivangi just completed, which added a few new properties.